pkg/asset/cluster/tfvars.go

```diff
@@ -338,16 +339,29 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 		}
 		preexistingnetwork := installConfig.Config.GCP.Network != ""
 
-		imageRaw, err := rhcospkg.GCPRaw(ctx, installConfig.Config.ControlPlane.Architecture)
+		archName := coreosarch.RpmArch(fmt.Sprintf("%s", installConfig.Config.ControlPlane.Architecture))
```

Comment:
Converting to a string works fine here.
```suggestion
		archName := coreosarch.RpmArch(string(installConfig.Config.ControlPlane.Architecture))
```